specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/customRecommedations.json

```diff
@@ -0,0 +1,384 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/customRecommendations": {
+      "get": {
+        "x-ms-examples": {
+          "List custom recommendations by subscription scope": {
+            "$ref": "./examples/CustomRecommendations/ListBySubscriptionCustomRecommendations_example.json"
+          },
+          "List custom recommendations by security connector scope": {
+            "$ref": "./examples/CustomRecommendations/ListBySecurityConnectorCustomRecommendations_example.json"
+          },
+          "List custom recommendations by management group scope": {
+            "$ref": "./examples/CustomRecommendations/ListByManagementGroupCustomRecommendations_example.json"
+          }
+        },
+        "tags": [
+          "CustomRecommendations"
+        ],
+        "description": "Get a list of all relevant custom recommendations over a scope",
+        "operationId": "CustomRecommendations_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CustomRecommendationsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
```

Comment:
>"../../../common/v2/types.json#/definitions/CloudError" [](http://example.com/codeflow?start=22&length=55)

please always consider using common type in ../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse

Reply:
fixed